modules/ixBidAdapter.js

```diff
@@ -560,32 +673,59 @@ export const spec = {
    * @return {boolean}     True if this is a valid bid, and false otherwise.
    */
   isBidRequestValid: function (bid) {
+    const paramsVideoRef = utils.deepAccess(bid, 'params.video');
+    const paramsSize = utils.deepAccess(bid, 'params.size');
+    const mediaTypeBannerSizes = utils.deepAccess(bid, 'mediaTypes.banner.sizes');
+    const mediaTypeVideoRef = utils.deepAccess(bid, 'mediaTypes.video');
+    const mediaTypeVideoPlayerSize = utils.deepAccess(bid, 'mediaTypes.video.playerSize');
+    const hasBidFloor = bid.params.hasOwnProperty('bidFloor');
+    const hasBidFloorCur = bid.params.hasOwnProperty('bidFloorCur');
+
     if (!isValidSize(bid.params.size)) {
       utils.logError('ix bidder params: bid size has invalid format.');
       return false;
     }
 
-    if (!includesSize(bid.sizes, bid.params.size)) {
-      utils.logError('ix bidder params: bid size is not included in ad unit sizes.');
+    if (bid.hasOwnProperty('mediaType') && !(utils.contains(SUPPORTED_AD_TYPES, bid.mediaType))) {
       return false;
     }
 
-    if (bid.hasOwnProperty('mediaType') && !(utils.contains(SUPPORTED_AD_TYPES, bid.mediaType))) {
+    if (bid.hasOwnProperty('mediaTypes') && !(mediaTypeBannerSizes || mediaTypeVideoPlayerSize)) {
       return false;
     }
 
-    if (bid.hasOwnProperty('mediaTypes') && !(utils.deepAccess(bid, 'mediaTypes.banner.sizes') || utils.deepAccess(bid, 'mediaTypes.video.playerSize'))) {
+    if (!includesSize(bid.sizes, bid.params.size) &&
```

Comment:
Use either `bid.params.size` or `paramsSize `. Don't mix both in the same `if ` condition